pkg/controller/pxc/secrets.go

```diff
@@ -96,20 +96,18 @@ func setUserSecretDefaults(secret *corev1.Secret) (isChanged bool, err error) {
 }
 
 const (
-	passwordMaxLen = 20
-	passwordMinLen = 16
-	passSymbols    = "ABCDEFGHIJKLMNOPQRSTUVWXYZ" +
+	passBaseSymbols = "ABCDEFGHIJKLMNOPQRSTUVWXYZ" +
 		"abcdefghijklmnopqrstuvwxyz" +
-		"0123456789" +
-		"!#$%&()*+,-.<=>?@[]^_{}~"
```

Comment:
By removing this we are essentially generating by default passwords without symbols which is actually the as is implementation. 

I think that we should not change the existing behaviour and have the existing symbols by default and only control which symbols are actually used or not by the cr option.